apps/web/app/api/v1/client/displays/[displayId]/responded/route.ts

```diff
@@ -27,3 +27,21 @@ export async function POST(_: Request, { params }: { params: { displayId: string
     return responses.internalServerErrorResponse(error.message);
   }
 }
+export async function PUT(
```

Comment:
As this is a normal display update, please move it to a new `api/v1/client/displays/[displayId]/route.ts` to be consistent with other routes.